### DISCUSSION TITLE
How to set up the citation key in Zotero Better BibTex to look this way... and also to be shown in Obsidian this way...

### DISCUSSION THREAD

**Discussion by iam8cosmiclove** [retorquere/zotero-better-bibtex#2488] [Q&A]
Hi there! I'm new to Zotero and Obsidian and I'm trying to set up the citation key in Zotero [to look this way](https://drive.google.com/file/d/17AslRd8QAOw3JdGDeptQhC3pKJxvTUIl/view?usp=sharing).

I've tried many things but couldn't make it work. 
The current configuration in Zotero is [this](https://drive.google.com/file/d/1y-T6J-cfk_VIOOqJ49rbs7RFHJvJwmCh/view?usp=sharing).
And it shows the field "Active citation key formula", so I guess it's wrong in some way. 

I put here another example of the current configuration in Zotero to show also how it looks then in Obsidian. It doesn't make sense to me. 
So, the current configuration in Zotero, selecting [another item](https://drive.google.com/file/d/1PaZrUcoLR39Jx6jBoXWx9FoYx3QdLiqB/view?usp=sharing), this time a movie with the director, two screenwriters and a producer. 
Here is [how it looks when I call this citation in Obsidian](https://drive.google.com/file/d/1eOwF4ZofF_wPL9lY4oAmAIhHnpEegWPx/view?usp=sharing). You can see here that it turns into @lastname1+3firstwordsoftitle+year. Where does this come from? How do I make it look like the desired configuration (first link). 

Thank you so much!!! If you need more information to explain myself, just tell me.

**Comment by retorquere** [comment]
Can you right-click that entry and select "send debug log" from the better bibtex menu? Something is not right. When the "active" field shows, I expect the entry field to be orange.

**Reply by iam8cosmiclove** [reply]
Done. :) What should I show you? The .txt? Is it safe to show it here? I don't understand the code :/

**Reply by retorquere** [reply]
Just the code.

**Comment by retorquere** [comment]
I don't see this discussion online anymore? The code will have `-apse` or
`-euc` at the end, that's all I need.
